content/enterprise_influxdb/v1.9/administration/monitor/_index.md

```diff
@@ -8,4 +8,23 @@ menu:
     weight: 50
 ---
 
+Monitoring is the act of observing changes in data over time.
+There are multiple ways to monitor your InfluxDB Enterprise cluster.
+See the guides below to monitor a cluster using another InfluxDB instance.
+
+Alternatively, to view your output data occasionally (_e.g._, for auditing or diagnostics),
+do one of the following:
+
+- [Log and trace InfluxDB Enterprise operations](/enterprise_influxdb/v1.9/administration/monitor/logs/)
+- [Use InfluxQL for diagnostics](/enterprise_influxdb/v1.9/administration/monitor/diagnostics/)
```

Comment:
Aren't these children docs? Do we need to list them separately?

Reply:
Yes. But I wanted to put them in context. They're actually _adjacent_ to monitoring, I'd say.

Reply:
Ok, that's fine.